src/components/implementation/tests/unit_defcompinfo/unit_defcompinfo.c

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright 2016, Phani Gadepalli and Gabriel Parmer, GWU, gparmer@gwu.edu.
+ *
+ * This uses a two clause BSD License.
```

Comment:
Do all of our unit tests use BSD, or do they use GPL?

I know the cos_kernel_api uses BSD, but that's different.

You're using code from the other file, so you have to make sure that the licences don't change without a significant social process.